content/partials/examples/dimmer.slim

```diff
@@ -1,63 +1,64 @@
-.dimmer.demo
-  .ui.segment
-    h3.ui.header
-      |  Dimmable Section
-    .ui.small.images
-      img.ui.wireframe.image[src="/static/kitchen-sink/images/wireframe/image.png"]
-      img.ui.wireframe.image[src="/static/kitchen-sink/images/wireframe/image.png"]
-      img.ui.wireframe.image[src="/static/kitchen-sink/images/wireframe/image.png"]
-    img.ui.medium.wireframe.image[src="/static/kitchen-sink/images/wireframe/media-paragraph.png"]
-    .ui.dimmer
-  .ui.ignored.ignored.icon.buttons
-    .ui.show.button
+.ui
```

Comment:
Another thing to supplement, the reason I use single ".ui" somewhere it that, it I use ".ui.button.demo" instead of ".ui.demo", the whole area will become a huge button, other parts also have the problem, like rating, sidebar.